test/cut/test_cut.py

```diff
@@ -37,3 +61,26 @@ def test_load_features(libri_cut):
     feats = libri_cut.load_features()
     assert feats.shape[0] == libri_cut.num_frames
     assert feats.shape[1] == libri_cut.features.num_features
+
+
+def test_load_none_features(libri_cut):
+    libri_cut.features = None
+    feats = libri_cut.load_features()
+    assert feats is None
+
+
+def test_make_cuts_from_recordings(libri_recording_set):
+    expected_duration = 16.04
+    cutset = make_cuts_from_recordings(libri_recording_set)
+    duration = list(cutset.cuts.values())[0].duration
```

Comment:
Note to self: we should probably make cutset work with both str (id) and int (index) based indexing

Reply:
Agree. The indexing by string(id) should not cost much.